provider/openstack/config_test.go

```diff
@@ -81,7 +81,7 @@ func (t configTest) check(c *gc.C) {
 		c.Assert(ecfg.FirewallMode(), gc.Equals, t.firewallMode)
 	}
 	c.Assert(ecfg.useDefaultSecurityGroup(), gc.Equals, t.useDefaultSecurityGroup)
-	c.Assert(ecfg.network(), gc.Equals, t.network)
+	c.Assert(ecfg.networks()[0], gc.Equals, t.network)
```

Comment:
is this better than
```
c.Check(ecfg.networks(), gc.DeepEquals, []string{t.network})
```
I do think this is a place where we should be using Check not Assert, but also matching the full list seems reasonable.

Reply:
Nice catch. This is vestigial from when I had it in the "vice" for changing the network config to a list.